vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.cc

```diff
@@ -53,12 +53,19 @@ void RefillTokens::Refill(
 
   public_key_ = public_key;
 
+  nonce_ = "";
```

Comment:
Seems unnecessary in the constructor, but fine by me if you'd just like to be explicit.

Reply:
We need the nonce to reset if starting without the need to retry

Reply:
Got it, I misread this as being the class's constructor. Sorry!